tasks/files-to-granules/index.js

```diff
@@ -0,0 +1,112 @@
+'use strict';
+
+const get = require('lodash.get');
+const path = require('path');
+
+const { getGranuleId } = require('@cumulus/cmrjs');
+const { parseS3Uri } = require('@cumulus/common/aws');
+const cumulusMessageAdapter = require('@cumulus/cumulus-message-adapter-js');
+
+/**
+ * Helper to turn an s3URI into a fileobject
+ * @param {string} s3URI s3://mybucket/myprefix/myobject.
+ * @returns {Object} file object
+ */
+function fileObjectFromS3URI(s3URI) {
+  const uriParsed = parseS3Uri(s3URI);
+  return {
+    name: path.basename(s3URI),
```

Comment:
There's a 99.999% chance that this will _never_ be an issue, but using `path` here will only work on systems that use a forward slash as a delimiter.  Probably not worth fixing since this project is targetted at Linux systems, but something to keep in mind.

Reply:
Good to know, but seems like a much larger issue than this one task, we use `path` everywhere.